### DISCUSSION TITLE
LAYOUT ISSUES: Github PR Review Dialog is not responsive

### DISCUSSION THREAD

**Discussion by regex5440** [community/community#113272] [Pull Requests]
## Unresponsive PR Review Dialog box:
> Github PR review dialog box is not responsive and the inner content gets overflowed from it, if the screen height is minimized.
[Screenshot]
>  <img width="665" alt="Screenshot 2024-03-19 at 5 03 54 PM" src="https://github.com/community/community/assets/34965581/bda0a6a5-216e-4bc2-bcb5-d06cdc8dc011">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @regex5440 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
